src/main/scala/mesosphere/marathon/MarathonSchedulerActor.scala

```diff
@@ -169,7 +169,6 @@ class MarathonSchedulerActor private (
       deploy(sender(), cmd)
 
     case cmd @ KillTasks(runSpecId, tasks) =>
-      @SuppressWarnings(Array("all")) /* async/await */
```

Comment:
You rock! When I read the title I was wondering whether you removed there or not.